README.md

```diff
@@ -13,19 +13,27 @@ A very simple package for accessing elements in the Periodic Table! :fire:
 
 ## Installation
 Since PeriodicTable is registered in `METADATA.jl`, you can directly install it like,
-```jl
+```julia
 julia> Pkg.add("PeriodicTable")
```

Comment:
Add `using Pkg` here.

Reply:
Would this seem better?
```julia
# for version <= 0.6
julia> using Pkg
julia> Pkg.add("PeriodicTable")

# for version >= 0.7
julia>] # switch to pkg> mode
(v1.0) pkg> add PeriodicTable
```

Reply:
No, the upper one won't work on 0.6 (there exists no module Pkg on 0.6). As I said, I'd just drop the `# for version ...` comments and only give instructions for 0.7+.

If you would really want to indicate both ways (0.6 and 0.7+) then I'd switch the order (0.7+ first) and remove the `using Pkg`.

Reply:
I missed that part.